styles.css

```diff
@@ -461,23 +474,157 @@ fieldset {
   margin: auto 27%;
 }
 
-@media (min-width: 768px) {
-  .hamberger-btn {
-    display: none;
-  }
+.drop-down {
+  position: absolute;
+  display: flex;
+  display: none;
+  flex-direction: column;
+  background-color: #3c3a39;
+  width: 100%;
+  height: 100%;
+}
 
-  #logo {
-    animation: rotate 1s ease-in-out forwards;
-  }
+#close-nav {
+  color: #fff;
+  display: flex;
+}
+
+.menu-link {
+  color: #fff5e1;
+  display: flex;
+  align-items: center;
+  padding: 10px;
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  text-decoration: none;
+  border-bottom-width: 1px;
+  border-bottom-style: solid;
+}
 
-  @keyframes rotate {
-    0% {
-      transform: rotate(0deg);
-    }
+a {
+  text-decoration: none;
+}
+
+#menu-close {
+  color: rgba(151, 148, 147, 1);
+  text-align: end;
+  padding: 4px;
+  height: 14px;
+}
+
+.dropdown-menu {
+  padding: 13px;
+}
+
+.dropdown-menuShow {
+  display: block;
+  z-index: 3;
+}
 
-    100% {
-      transform: rotate(360deg);
-    }
+/* Styles for the popup */
+.project-card {
+  padding: 20px;
+  margin-bottom: 20px;
+  cursor: pointer;
+}
+
+#modalOverlay {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 9999;
+}
+
+#modalTitle {
+  color: #172b4d;
+  text-align: start;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+#modalContent {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  background-color: #fff;
+  padding: 20px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 90%;
+  transform: translate(-50%, -50%);
+}
+
+#modalImage {
+  width: inherit;
+  display: block;
+  height: 220px;
+}
+
+#closeModalBtn {
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+.technologies-list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+}
+
+.technologies-list li {
+  border: 1px solid #979493;
+  padding: 10px 12px;
+  width: auto;
+  height: 14px;
+}
+
+#modalDescription {
+  width: inherit;
+  color: #344563;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+#linkflex {
+  display: flex;
+  flex-direction: row;
+  gap: 27px;
+}
+
+#linkflex a {
+  background: #ff6b00;
+  width: 125px;
+  height: 24px;
+  padding: 10px;
+  color: #fff;
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+}
```

Comment:
- [ ] Kindly stict to the figma template design ,the see `live version` and the `source code` button should be place at left side under the text content kindly fix this to make your website look like original figma template and profesional.
- [ ] figma desgn
![image](https://github.com/Obimbo07/Portfolio/assets/92458236/7e70989d-efa8-4171-a296-7c00390bd729)

- [ ] your design 
![zzz](https://github.com/Obimbo07/Portfolio/assets/92458236/6156e976-7486-49eb-907d-332f171a9821)